src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionSettings.cs

```diff
@@ -23,6 +23,8 @@ internal sealed class HttpConnectionSettings
         internal bool _useProxy = HttpHandlerDefaults.DefaultUseProxy;
         internal IWebProxy _proxy;
         internal ICredentials _defaultProxyCredentials;
+        internal bool _defaultSystemCredentialsUsedForProxy;
+        internal bool _defaultSystemCredentialsUsedForServer;
```

Comment:
nit: The term "defaultSystemCredentials" is a new word we haven't used before.  I prefer to keep the "defaultCredentials" prefix on these fields since it is obvious what "defaultCredentials" are (i.e. CredentialCache.DefaultCredentials, etc.)

So, I would prefer to see:

```c#
internal bool _defaultCredentialsUsedForProxy;
internal bool _defaultCredentialsUsedForServer;
```